### ISSUE TITLE
'*' in eix-sync.conf no longer syncs layman, expands to files in current directory

### ISSUE THREAD

**Issue by mittwinter** [vaeth/eix#29]
I have a single asterisk in the file /etc/eix-sync.conf which should result in layman -S being called as per the documentation. Hovever as of eix version 0.31.8 this no longer happens, instead it expands to files in the current working directory:

``` bash
$ eix -V
0.31.8
$ touch a_file
$ cat /etc/eix-sync.conf
*
$ eix-sync
 * Syncing portage overlay a_file

 * Fetching remote list...
 * Fetch Ok

 * Syncing selected overlay(s)...
Traceback (most recent call last):
  File "/usr/lib/python-exec/python2.7/layman", line 45, in <module>
    main()
  File "/usr/lib64/python2.7/site-packages/layman/cli.py", line 196, in __call__
    result += getattr(self, action[1])()
  File "/usr/lib64/python2.7/site-packages/layman/cli.py", line 281, in Sync
    result = self.api.sync(selection, update_news=True)
  File "/usr/lib64/python2.7/site-packages/layman/api.py", line 401, in sync
    % {'repo': ovl, 'err': error}))
KeyError: u'error'
 * layman -s a_file failed
 * Syncing portage overlay <next_file>
[...]
```

**Comment by vaeth** [comment]
This was a regression from a Bourne shell (also of some ksh versions)  incompatibility workaround.
The issue is (hopefully) fixed in eix-0.31.9 which has been released today.
See also https://bugs.gentoo.org/show_bug.cgi?id=591422
